sui-execution/latest/sui-verifier/src/id_leak_verifier.rs

```diff
@@ -409,14 +427,14 @@ fn execute_inner(
 
         // These bytecodes produce references, and hence cannot be ID.
         Bytecode::MutBorrowLoc(_)
-        | Bytecode::ImmBorrowLoc(_) => verifier.stack.push(AbstractValue::Other),
+        | Bytecode::ImmBorrowLoc(_) => verifier.stack_push(AbstractValue::Other).unwrap(),
```

Comment:
Any reason this should be an unwrap as opposed to a `?`

Reply:
I think I missed just switching this to ? (which wasn't possible in an earlier version of this PR)